packages/vscode-lit-plugin/copy-to-built.js

```diff
@@ -13,7 +13,10 @@ const { copy, mkdirp, writeFile } = require("fs-extra");
 async function main() {
 	// We don't bundle the typescript compiler into ./built/bundle.js, so we need
 	// a copy of it.
-	await copy("./node_modules/typescript", "./built/node_modules/typescript");
+	await mkdirp("./node_modules/typescript/lib");
+	await copy("./node_modules/typescript/package.json", "./built/node_modules/typescript/package.json");
+	await copy("./node_modules/typescript/lib/typescript.js", "./built/node_modules/typescript/lib/typescript.js");
```

Comment:
we don't need any of the libs?

Reply:
It seems like no, we don't

I wish that we didn't need typescript here at all, because we only use it in a marginal way via lit analyzer to find html templates to find styles to find colors in those styles to tell vscode about them